.ci/install_kata_kernel.sh

```diff
@@ -55,6 +55,12 @@ build_and_install_kernel() {
 	"${kernel_packaging_dir}/build-kernel.sh" -v "${kernel_version}" ${extra_opts} "setup"
 	"${kernel_packaging_dir}/build-kernel.sh" -v "${kernel_version}" ${extra_opts} "build"
 	sudo -E PATH="$PATH" "${kernel_packaging_dir}/build-kernel.sh" -v "${kernel_version}" ${extra_opts} "install"
+
+	### REMOVE after kata-containers PR #5010 is merged
```

Comment:
5010 is closed so let's update this comment.

Reply:
Needed to change this to 5013. Updated the code comment.